e2e_deployment_files/long_haul_deployment.template.json

```diff
@@ -208,78 +208,6 @@
               "image": "influxdb:1.7",
               "createOptions": "{\"ExposedPorts\":{\"8086/tcp\":{}},\"HostConfig\":{\"PortBindings\":{\"8086/tcp\":[{\"HostPort\":\"8086\"}]}}}"
             }
-          },
```

Comment:
Are this removed for all environments, should it be for arm only?

Reply:
create a new long haul deployment file for arm